code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -624,13 +624,18 @@
 	desc = "A special box for sensitive people."
 	icon_state = "hugbox"
 	foldable = null
+	var/cooldown = 0
 
 /obj/item/weapon/storage/box/hug/attack_self(mob/user)
-	..()
-	user.changeNext_move(CLICK_CD_MELEE)

```

Comment:
Okay, I'll see if I can figure it out.

On 21 July 2015 at 17:00, Ikarrus <notifications@github.com> wrote:

> In code/game/objects/items/weapons/storage/boxes.dm
> <https://github.com/tgstation/-tg-station/pull/10734#discussion_r35069900>
> :
>
> >
> >  /obj/item/weapon/storage/box/hug/attack_self(mob/user)
> > -	..()
> > -	user.changeNext_move(CLICK_CD_MELEE)
>
> If I haven't made it clear, that proc is supposed to be a cooldown proc,
> to prevent spam/atatatata attacks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/10734/files#r35069900>.
>